### ISSUE TITLE
Weird mic and refresh icon on the bottom right

### ISSUE THREAD

**Issue by mrkalmdn** [microsoft/terminal#10545]
### Windows Terminal version (or Windows build number)

1.8.1521.0

### Steps to reproduce

It just shows up sometimes.

### Actual Behavior

![image](https://user-images.githubusercontent.com/10385316/124224547-67bca580-db38-11eb-9a78-7ed81224993c.png)

**Comment by skyline75489** [comment]
Is there any other software you're using? I'm pretty sure these icons do not belong to the terminal.

**Comment by mrkalmdn** [comment]
@skyline75489 I see. Currently running softwares are:

- PowerToyz
- VS Code
- Notion
- Edge

**Comment by mrkalmdn** [comment]
Will close the issue. Thanks!

**Comment by ThomasMConnors** [comment]
This was driving me crazy! I tracked this problem down to the NVIDIA GForce Experience.

Seems that GForce thinks the WinTerm Preview is a video game and you might want to record your session and comment on it.

I just stopped the NVIDIA services and set them to manual.

MS, you might want to investigate why GForce Experience thinks this is a game or what you have enabled in the Preview build that looks like a game.
